Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -70,6 +71,8 @@ class OWTestLearners(widget.OWWidget):
     #: Bootstrap sampling p
     sample_p = settings.Setting(75)
 
+    class_selection = settings.Setting("(None)")
```

Comment:
This needs to be a context setting. Use ClassValuesContextHandler; if it doesn't work, as @astaric whether he's fixing it.